apps/web-giddh/src/app/expenses/components/expense-details/expense-details.component.html

```diff
@@ -5,13 +5,24 @@
                 <p class="bold mr-b05">{{ selectedItem?.entryDate }} | {{ selectedItem.baseAccount.name }}</p>
             </div>
             <div class="col-sm-6 text-right xs-left">
-                <button type="button" class="mr-b05 btn btn-success" (click)="showApproveConfirmPopup(ApproveConfirm)">
+                <button
+                    mat-raised-button
+                    color="primary"
+                    type="button"
+                    (click)="showApproveConfirmPopup(approveConfirm)"
+                >
                     {{ localeData?.approve }}
                 </button>
-                <button type="button" class="mr-b05 btn btn-danger" (click)="openModal(RejectionReason)">
+                <button
+                    mat-raised-button
+                    color="accent"
+                    type="button"
+                    class="mr-l1"
```

Comment:
remove the color and use class name properly. @ravinderwalkover

Reply:
Done